combat-trainer.lic

```diff
@@ -2215,6 +2215,11 @@ class TrainerProcess
     when 'App Bundle'
       retreat
       bput("app my bundle quick", 'You scan', 'Appraise what', 'You cannot appraise', 'You can\'t appraise')
+    when /Summon (?<element>Aether|Air|Earth|Electricity|Fire|Water)/
+      retreat
+      bput("summon #{Regexp.last_match[:element]}", 'turn your attention inward', 'Roundtime')
```

Comment:
> It is possible to change domains.

Is there different text for changing domains, vs making a new one?

Reply:
I don't think there's different texts. Here's output when I try to summon domain for each element:

```
> summon aether
You make yourself comfortable and turn your attention inward.
Drawing down extraplanar laws, you attempt to create an aether domain spanning the area.
As the ritual winds toward its climax, you realize that you do not have the Elemental Charge to create an aether domain.
Roundtime: 10 sec.

> summon air
You make yourself comfortable and turn your attention inward.
Drawing down extraplanar laws, you attempt to create an air domain spanning the area.
As the ritual winds toward its climax, you realize that you do not have the Elemental Charge to create an air domain.
Roundtime: 10 sec.

> summon earth
You make yourself comfortable and turn your attention inward.
Drawing down extraplanar laws, you attempt to create an earth domain spanning the area.
As the ritual winds toward its climax, you realize that you do not have the Elemental Charge to create an earth domain.
Roundtime: 10 sec.

> summon electricity
You make yourself comfortable and turn your attention inward.
Drawing down extraplanar laws, you attempt to create an electricity domain spanning the area.
As the ritual winds toward its climax, you realize that you do not have the Elemental Charge to create an electricity domain.
Roundtime: 10 sec.

> summon fire
You make yourself comfortable and turn your attention inward.
Drawing down extraplanar laws, you attempt to create a fire domain spanning the area.
As the ritual winds toward its climax, you realize that you do not have the Elemental Charge to create a fire domain.
Roundtime: 10 sec.

> summon water
You make yourself comfortable and turn your attention inward.
Drawing down extraplanar laws, you attempt to create a water domain spanning the area.
As the ritual winds toward its climax, you realize that you do not have the Elemental Charge to create a water domain.
Roundtime: 10 sec.
```

Reply:
Confirmed, we can continue to match on "turn your attention inward" even when summoning a different domain.

```
s> summon fire domain

You make yourself comfortable and turn your attention inward.
Drawing down extraplanar laws, you attempt to create a fire domain spanning the area.
Discharging a burst of elemental power in the esoteric "direction" of the Elemental Plane of Fire, you carefully draw down subtle lines of power across the Void.
The strands anchor themselves to the ground, infusing the area with an additional, subordinate set of physical laws that make fire magic easier.
Roundtime: 10 sec.

[multi]>summon admit

You make yourself comfortable and turn your attention inward.
Tracing the lines of the aetheric landscape, you find a transient weak point.  You align yourself to it, briefly increasing your proximity to the Elemental Plane of Aether.
Roundtime: 15 sec.

... summon admit a few more times to build elemental charge ...

s> summon air domain
You make yourself comfortable and turn your attention inward.
Drawing down extraplanar laws, you attempt to create an air domain spanning the area.
Discharging a burst of elemental power in the esoteric "direction" of the Elemental Plane of Air, you carefully draw down subtle lines of power across the Void.
The new lines of power are in conflict with the existing domain!  The existing lines hold, though weakened.
Roundtime: 10 sec.
```